mslice/models/slice/slice_algorithm.py

```diff
@@ -57,8 +57,9 @@ def _compute_slice_PSD(self, workspace, x_axis, y_axis, norm_to_one):
         y_dim = workspace.getDimension(y_dim_id)
         xbinning = x_dim.getName() + "," + str(x_axis.start_meV) + "," + str(x_axis.end_meV) + "," + str(n_x_bins)
         ybinning = y_dim.getName() + "," + str(y_axis.start_meV) + "," + str(y_axis.end_meV) + "," + str(n_y_bins)
-        return BinMD(InputWorkspace=workspace, AxisAligned="1", AlignedDim0=xbinning, AlignedDim1=ybinning,
-                     StoreInADS=False)
+        thisslice = BinMD(InputWorkspace=workspace, AxisAligned="1", AlignedDim0=xbinning, AlignedDim1=ybinning,
+                          StoreInADS=False)
+        return thisslice
```

Comment:
This will create a workspace in the ADS called `thisslice`. Do we still want this, or was it for testing purposes?

Reply:
Given  `StoreInADS=False`, why would it create a workspace in the ADS? I needed to put this line in rather than returning the output of `BinMD` directly to make a test work.

Reply:
Ah, didn't see `StoreInADS=False` 👍